src/debugger/Debugger.js

```diff
@@ -170,9 +169,9 @@ export class DebugServer {
     return false;
   }
 
-  processStackframesCommand(requestID: number, args: StackframeArguments, ast: BabelNode) {
+  processStackframesCommand(requestID: number, args: StackframeArguments, ast?: BabelNode) {
```

Comment:
This should be "ast: void | BabelNode". "ast?:" tells Flow not to complain if the ast argument is not supplied.